cmd/neofs-cli/modules/container/get_eacl.go

```diff
@@ -30,7 +31,11 @@ var getExtendedACLCmd = &cobra.Command{
 
 		if containerPathTo == "" {
 			cmd.Println("eACL: ")
-			common.PrettyPrintJSON(cmd, &eaclTable, "eACL")
+			if containerJSON {
```

Comment:
It seems unrelated, let's do it in a different commit?

Reply:
Reverted.